src/participant_network.star

```diff
@@ -88,7 +88,7 @@ def launch_participant_network(plan, participants, network_params, global_log_le
 			fail("Unsupported launcher '{0}', need one of '{1}'".format(el_client_type, ",".join([el.name for el in el_launchers.keys()])))
 
 		el_launcher, launch_method = el_launchers[el_client_type]["launcher"], el_launchers[el_client_type]["launch_method"]
-		el_service_name = "{0}-{1}-{2}".format(index+1, el_client_type, cl_client_type)
```

Comment:
can you suffix `svc` instead? that way we get the right ordering in the kurtosis enclave inspect ui!